index.md

```diff
@@ -18,10 +18,14 @@ The WSTG is a comprehensive guide to testing the security of web applications an
 
 ## Stable
 
-View the always-current stable version at [/stable](/www-project-web-security-testing-guide/stable).
+View the always-current stable version at [/stable](stable/).
```

Comment:
Is the link correct? `stable/` and not `/stable`?

Same for `latest` and `v41`.

Reply:
Yes.

Reply:
It's relative to index.md, tab_download, etc

I guess we could do it as ./stable/ but that seems unnecessary.

Also the trailing slash is necessary to avoid a redirect, as seen here:
![image](https://user-images.githubusercontent.com/7570458/78505588-74d6c300-7742-11ea-9333-7c869c0a6564.png)